modules/stanford_news_views/stanford_news_views.views_default.inc

```diff
@@ -1264,7 +1269,7 @@ function stanford_news_views_views_default_views() {
   );
   $handler->display->display_options['filters']['field_s_news_categories_tid']['type'] = 'select';
   $handler->display->display_options['filters']['field_s_news_categories_tid']['vocabulary'] = 'news_categories';
-  $handler->display->display_options['path'] = 'news/recent-news';
+  $handler->display->display_options['path'] = 'news/in-the-news';
```

Comment:
This page display has a different url change than the other ones. Was this intentional? Does it need to be added to the menu somewhere?